helm-chart/renku/templates/post-install-job-keycloak.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.keycloak.enabled }}
```

Comment:
I'm a bit confused: if you're adding an `if` shouldn't be there also an `end`?

Reply:
yes, good catch. In the rush I forgot to push it.
Should be ok now :)